src/test/java/edu/iis/mto/similarity/SimilarityFinderTest.java

```diff
@@ -1,14 +1,133 @@
 package edu.iis.mto.similarity;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.fail;
 
+import edu.iis.mto.searcher.SearchResult;
+import edu.iis.mto.searcher.SequenceSearcher;
 import org.junit.jupiter.api.Test;
 
 class SimilarityFinderTest {
 
+    public int counter;
+
+    SequenceSearcher sequenceSearcher = new SequenceSearcher() {
+        @Override
+        public SearchResult search(int elem, int[] sequence) {
+            for (int i = 0; i < sequence.length; i++) {
```

Comment:
nie chodziło o to, żeby implementować algorytm wyszukiwania - to miał być "mock"